rhui_provision/rhui_cloud_formation.template

```diff
@@ -202,6 +202,67 @@
               ]
             }
         }, 
+        "monitoringService": {
+            "Type": "AWS::EC2::Instance", 
+            "Properties": {
+                "Tags": [
+                    {
+                        "Value": {
+                            "Fn::Join": [
+                                "_", 
+                                [
+                                    {
+                                        "Ref": "OwnerName"
+                                    },
+                                    "monitoringService"
+                                ]
+                            ]
+                        }, 
+                        "Key": "Name"
+                    }, 
+                    {
+                        "Value": "MON_SER", 
+                        "Key": "Role"
+                    } 
+                ], 
+                "KeyName": {
+                    "Ref": "KeyName"
+                }, 
+                "SecurityGroups": [
+                    {
+                        "Ref": "Monitoringsecuritygroup"
+                    }
+                ], 
+                "InstanceType": { "Ref": "InstanceType"}, 
+                "ImageId": {
+                    "Fn::FindInMap": [
+                        "RHEL64", 
+                        {
+                            "Ref": "AWS::Region"
+                        }, 
+                        "AMI"
+                    ]
+                },
+                "BlockDeviceMappings": [
```

Comment:
I don't think we need all of these disks for a monitoring instance. We set up separate partitions for /var/log, /var/lib/mongodb, /var/lib/pulp on RHUI instances.